.github/workflows/expo-eas-clean.yml

```diff
@@ -0,0 +1,18 @@
+name: expo-eas-clean
```

Comment:
to clean the eas update list after the PR is closed.

Reply:
I thought this was done automatically!

Reply:
I'm fairly confident that before eas they were being deleted automatically, I mean by expo, not us